drivers/virtio/ring/Makefile.uk

```diff
@@ -4,6 +4,7 @@ $(eval $(call addlib_s,libvirtio_ring,$(CONFIG_LIBVIRTIO_RING)))
 LIBVIRTIO_RING_CINCLUDES-y  += -I$(UK_DRIV_LIBVIRTIO_BASE)/include
 LIBVIRTIO_RING_CINCLUDES-y  += -I$(LIBVIRTIO_BUS_BASE)/include
 LIBVIRTIO_RING_CINCLUDES-y  += -I$(LIBVIRTIO_RING_BASE)/include
+LIBVIRTIO_RING_CINCLUDES-y  += -I$(LIBVIRTIO_BUS_BASE)/include
```

Comment:
I believe this is a duplicate of line 5, might've slipped in through some rebase; we should remove it.

Reply:
I just checked: It got added in PR #1095. It's not very clear why that PR also added the include.